code_generator/GdkCodeGenerator/Templates/UnityCommandPayloadGenerator.tt

```diff
@@ -85,8 +85,8 @@ namespace <#= qualifiedNamespace #>
                 public EntityId EntityId { get; }
                 public string Message { get; }
                 public StatusCode StatusCode { get; }
-                public <#= commandDetails.ResponseType #>? RawResponse { get; }
-                public <#= commandDetails.RequestType #> RawRequest { get; }
+                public <#= commandDetails.ResponseType #>? Payload { get; }
```

Comment:
Could we call this `ResponsePayload`? Or if we just call it payload then we could name the request `CorrespondingRequest`.

Reply:
Wild alternative is to call the `Reponse` `CommandResponse` and then have a `Response` as a field. I'm mostly just against using raw in the name for this case. If we can't decide on something then I'm happy to approve this though.

Reply:
I'm happy with `Payload` and `CorrespondingRequest` if you are